### DISCUSSION TITLE
Change Language settings

### DISCUSSION THREAD

**Discussion by Perkaplex** [SafeExamBrowser/seb-win-refactoring#191] [Q&A]
We use SEB on several PCs all with the same settings:
Windows 10 Pro (Build 19042.1165)
SEB Version: 3.2.0.324
Systemlanguage on every PC is german, but browserlanguage in SEB is still english.
There are no settings in SEBConfigTool to choose a specific language. How do I change the language from english to german?
Hope someone can help me.

**Comment by dbuechel** [comment]
Thanks for the hint, we'll track this issue with #194.
